Volha_Ivanova/Module_1_Java/Java_Fundamentals/HomeWork1/src/com/epam/cdp/volha_ivanova_task1/Runner.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.cdp.volha_ivanova_task1;
+
+public class Runner {                       //Volha Ivanova. В массиве целых чисел определить сумму элементов, состоящих на чётных позициях
+
+	int [] arr1;
+			
+	public static void main(String[] args) {
+    int [] arr1 = new int [20];             // declare massive and its length
+    int min = -10;
+    int max = 10;
+    int j = min;
+    
+   System.out.println("Array elements: \n");
+   
+    for (int i=0; i<arr1.length; i++) {     // filling massive
+    	while (j <= max) { 
+    		if (j!=0) {
+    	arr1[i] = j;
+    	j++;
+    System.out.println(arr1[i]);
+    	}  	else {j++;}
+    }
+    }
```

Comment:
I think it's very complex logic for filling array. And a lot of useless variables
You can use 
` Random rand = new Random();
        for (int i = 0; i < arr1.length; i++)
        {
            arr1[i] = rand.nextInt(max);
        }`